themes/default/content/registry/packages/aws/how-to-guides/bucketv2-migration.md

```diff
@@ -0,0 +1,2423 @@
+---
+title: "BucketV2 Migration Guide"
+h1: "Migrating from aws.s3.Bucket to aws.s3.BucketV2"
+meta_desc: Practitioner level instructions for migrating from aws.s3.Bucket to aws.s3.BucketV2 resources
+layout: package
+---
+
+In the upcoming AWS Classic major release (v7), `aws.s3.Bucket` will be discontinued in favor of `BucketV2`. Users of
+`aws.s3.Bucket` resource are encouraged to migrate early. The migration is straightforward for simple use cases but
+requires some care for advanced configurations. Specifically `BucketV2` inputs such as `policy` and `accelerationStatus`
+are deprecated and using the requires migrating to side-by-side resources (see Removed inputs). This guide aims to cover
+all relevant scenarios with precise migration instructions.
+
+To migrate existing `aws.s3.Bucket` resources to `aws.s3.BucketV2`:
+
+1. Edit your Pulumi program sources to replace removed inputs (see below) with equivalent side-by-side resources.
+
+2. Perform `pulumi up` to replace (delete and re-create) the buckets in AWS.
+
+If replacement is not acceptable, it is possible to perform a manual migration with `pulumi import` (see Avoiding
+replacement).
```

Comment:
We should be able to link to the section like this: `[Avoiding Replacement](#avoiding-replacement)`. Not 100% sure though whether Hugo supports that syntax

Reply:
It's been failing builds. Looks like this only works in GitHub markdown. @interurban might have some pointers here.